Makefile

```diff
@@ -1982,20 +1975,18 @@ TEST_APPS=\
 TEST_APPS_DEPS=$(TEST_APPS:%=%_test_app)
 BUILD_APPS_DEPS=$(TEST_APPS:%=%_build_app)
 
-$(BUILD_APPS_DEPS): distrib build_python_bindings
```

Comment:
I think some of the apps may require the python bindings... maybe onnx? Not sure

Reply:
Yeah, I said in the PR description that apps/onnx and apps/HelloPyTorch might break. Was going to wait and see.